Tools/TestWebKitAPI/Tests/WebKitCocoa/WKAttachmentTests.mm

```diff
@@ -146,6 +146,29 @@ - (void)_webView:(WKWebView *)webView didInvalidateDataForAttachment:(_WKAttachm
 
 @end
 
+static NSURL *testiWorkAttachmentFileURL()
```

Comment:
Need to remove the re-definition of this function further down to fix the build errors, this got lost in the rebase.